pkg/common-controller/framework_test.go

```diff
@@ -305,6 +406,46 @@ func (r *snapshotReactor) React(action core.Action) (handled bool, ret runtime.O
 		klog.V(4).Infof("saved updated content %s", content.Name)
 		return true, content, nil
 
+	case action.Matches("patch", "volumegroupsnapshotcontents"):
+		content := &crdv1alpha1.VolumeGroupSnapshotContent{}
+		action := action.(core.PatchAction)
+
+		// Check and bump object version
+		storedGroupSnapshotContent, found := r.groupContents[action.GetName()]
+		if found {
+			// Apply patch
+			storedSnapshotBytes, err := json.Marshal(storedGroupSnapshotContent)
```

Comment:
storedSnapshotBytes to storedGroupSnapshotBytes

Reply:
Fixed!